docs/about/index.html

```diff
@@ -24,7 +24,7 @@
 
     <link rel="stylesheet" href="https://packit.dev/css/style.css">
 
-    <meta name="generator" content="Hugo 0.55.0" />
+    <meta name="generator" content="Hugo 0.54.0" />
```

Comment:
`dnf update` please

Reply:
I downloaded random library from the internet, lol :D